package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "start": "cross-env NODE_ENV=development webpack-dev-server --hot --inline --watch --content-base dist --port 8085",
-    "dist": "cross-env NODE_ENV=production webpack -p --config webpack.production.config.js",
+    "dist": "cross-env NODE_ENV=production webpack --config webpack.production.config.js",
```

Comment:
Removing `-p` removes `--optimize-minimize` and `--optimize-occurence-order` per https://webpack.github.io/docs/cli.html#production-shortcut-p

I see you add uglify back in below, but not occurrence order (dedupe isn't the same) - and I was wondering why we don't want occurrence order anymore? (Curious)